src/traveltime_google_comparison/requests/here_handler.py

```diff
@@ -62,6 +62,14 @@ async def send_request(
                         for section in first_route["sections"]
                     )
 
+                    # For some reason, HERE provider returns 0 duration, 0 length
+                    # for some routes in the mountains, but doesn't indicate anywhere
+                    # that it failed. Returning 0 fails `asType(int)` conversion later.
+                    # Example route in UK where this happens:
+                    # "58.61966879999991, -5.0040819999999995","58.578906999999894, -4.880025099999999"
+                    if total_duration == 0:
+                        return RequestResult(None)
+
```

Comment:
With the whole UK input file, this only happened for the HERE provider, nowhere else.

Another solution would be to simply remove the locations (only 4 of them) that return this 0 travel time result. They seem to get pretty random results in other APIs, ranging from 900 to 7500. But even if we do that, having this for safety is still probably nice.